node/src/components/network.rs

```diff
@@ -49,26 +63,19 @@ use crate::{
     utils::DisplayIter,
     NodeRng,
 };
-pub use config::Config;
-pub use error::Error;
-pub(crate) use event::Event;
-pub(crate) use message::Message;
-
-/// The timeout for connection setup (including upgrades) for all inbound and outbound connections.
-// TODO - make this a config option.
-const CONNECTION_TIMEOUT: Duration = Duration::from_secs(10);
+
 /// Env var which, if it's defined at runtime, enables the libp2p server.
 pub(crate) const ENABLE_LIBP2P_ENV_VAR: &str = "CASPER_ENABLE_LIBP2P";
```

Comment:
I'll leave this as is rather than change it, to shortly remove it altogether.

I've been trying to leave the configuration untouched until we come to removing the `small_network` component.

Reply:
Fair enough :)